crates/nix_rs/src/flake/schema.rs

```diff
@@ -16,12 +16,14 @@ use super::{
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct FlakeSchema {
     pub system: System,
-    pub packages: BTreeMap<String, Val>,
-    pub legacy_packages: BTreeMap<String, Val>,
-    pub devshells: BTreeMap<String, Val>,
-    pub checks: BTreeMap<String, Val>,
-    pub apps: BTreeMap<String, Val>,
-    pub formatter: Option<Val>,
+    pub packages: BTreeMap<String, Leaf>,
+    pub legacy_packages: BTreeMap<String, Leaf>,
+    pub devshells: BTreeMap<String, Leaf>,
+    pub checks: BTreeMap<String, Leaf>,
+    pub apps: BTreeMap<String, Leaf>,
+    pub formatter: Option<Leaf>,
+    pub nixos_configurations: BTreeMap<String, Leaf>,
+    pub darwin_configurations: BTreeMap<String, Leaf>,
```

Comment:
Missing from the list:

- `nixos_modules`
- `homeConfigurations`
- `dockerImages`
- `overlays`
- `templates`
- `schemas`

https://github.com/DeterminateSystems/flake-schemas/blob/29e53dd33b1a38f235ef073e768c62821cb6146e/flake.nix#L333-L347

Reply:
Added in https://github.com/juspay/omnix/pull/166/commits/ccc233a8d45568c7ac73954fd8c84b820bfad07a:

See a few examples...
### Templates
```sh
❯ cargo run -p omnix-cli -- show github:srid/haskell-flake
📝 Templates

Run: nix flake init -t github:srid/haskell-flake#<name>
─────────┬─────────────────────────────────────────────
 name    │ description
═════════┼═════════════════════════════════════════════
 default │ A simple flake.nix using haskell-flake
 example │ Example Haskell project using haskell-flake
─────────┴─────────────────────────────────────────────
```

### Schemas

```sh
❯ cargo run -p omnix-cli -- show github:DeterminateSystems/flake-schemas
📜 Schemas

Run:
──────────────────────┬──────────────────────────────────────────────────────────────
 name                 │ description
══════════════════════┼══════════════════════════════════════════════════════════════
 apps                 │ A schema checker for the `apps` flake output
 checks               │ A schema checker for the `checks` flake output
 darwinConfigurations │ A schema checker for the `darwinConfigurations` flake output
 devShells            │ A schema checker for the `devShells` flake output
 dockerImages         │ A schema checker for the `dockerImages` flake output
 formatter            │ A schema checker for the `formatter` flake output
 homeConfigurations   │ A schema checker for the `homeConfigurations` flake output
 hydraJobs            │ A schema checker for the `hydraJobs` flake output
 legacyPackages       │ A schema checker for the `legacyPackages` flake output
 nixosConfigurations  │ A schema checker for the `nixosConfigurations` flake output
 nixosModules         │ A schema checker for the `nixosModules` flake output
 overlays             │ A schema checker for the `overlays` flake output
 packages             │ A schema checker for the `packages` flake output
 schemas              │ A schema checker for the `schemas` flake output
 templates            │ A schema checker for the `templates` flake output
──────────────────────┴──────────────────────────────────────────────────────────────
```

### nixosModules

```sh
❯ cargo run -p omnix-cli -- show github:srid/nixos-config
...
🔧 NixOS Modules

Run:
──────────────┬─────────────
 name         │ description
══════════════┼═════════════
 common       │ N/A
 default      │ N/A
 home-manager │ N/A
 my-home      │ N/A
 nixosFlake   │ N/A
──────────────┴─────────────
```